tests/e2e/support/objects/app-files/resource/actions.ts

```diff
@@ -93,6 +95,16 @@ const pauseResumeUploadButton = '#pause-upload-info-btn'
 const cancelUploadButton = '#cancel-upload-info-btn'
 const uploadPauseTooltip = '//div[text()="Pause upload"]'
 const uploadResumeTooltip = '//div[text()="Resume upload"]'
+const savedInCollaboraEditorSelector = '.notebookbar-shortcuts-bar .savemodified'
+const onlyOfficeInnerFrameSelector = '[name="frameEditor"]'
+const microsoftWordSaveButtonSelector = '#asc-gen578'
+const openDocumentTextAreaSelector = '#clipboard-area'
+const microsoftWordTextAreaSelector = '#area_id'
```

Comment:
I think, the selector is different for different editor not for different doc format.
so, this could be good way:
```suggestion
const collaboraDocTextAreaSelector = '#clipboard-area'
const onlyofficeDocTextAreaSelector = '#area_id'
```